pyproject.toml

```diff
@@ -217,6 +218,9 @@ filterwarnings = [
   'ignore:.*The distutils package is deprecated and slated for removal in Python 3.12:DeprecationWarning:pyspark',
   'ignore:.*distutils Version classes are deprecated. Use packaging.version instead.*:DeprecationWarning:pyspark',
   'ignore:.*is_datetime64tz_dtype is deprecated and will be removed in a future version.*:DeprecationWarning:pyspark',
+  'ignore:.*No Partition Defined for Window operation! Moving all data to a single partition.*:UserWarning:pyspark',
```

Comment:
This looks quite bad honestly! I will try to do some digging or if anyone knows :)